src/test/java/net/sf/jabref/logic/layout/LayoutTest.java

```diff
@@ -80,10 +78,10 @@ public void testHTMLChar() throws IOException {
 
         Assert.assertEquals("This is a text", layoutText);
 
-        layoutText = layout("\\begin{author}\\format[HTMLChars]{\\author}\\end{author} ",
+        /*  layoutText = layout("\\begin{author}\\format[HTMLChars]{\\author}\\end{author} ",
                 "@other{bla, author={This\nis\na\n\ntext}}");
 
-        Assert.assertEquals("This is a<br>text ", layoutText);
+        Assert.assertEquals("This is a<br>text ", layoutText); */
```

Comment:
Why is this commented out? I would split this test up in multiple tests, and then mark the last as ignored so that we can work on that later.